examples/wind/src/control-panel.js

```diff
@@ -47,7 +47,7 @@ export default class ControlPanel extends Component {
         { this._renderToggle('showParticles', 'particles') }
         { this._renderToggle('showWind', 'field') }
         { this._renderToggle('showElevation', 'elevation') }
-        { this._renderToggle('useDevicePixels', 'Retina/HD') }
+        { this._renderToggle('useDevicePixels', 'Retina/ HD') }
```

Comment:
why do we need this space?

Reply:
Just a non functional change, it looked better this way.